modules/seqexec/web/client/src/main/scala/seqexec/web/client/reusability/package.scala

```diff
@@ -104,4 +95,20 @@ package object reusability {
     Reusability.derive[M2GuideConfig]
   implicit val configReuse: Reusability[TelescopeGuideConfig] =
     Reusability.derive[TelescopeGuideConfig]
+  val stdStepReuse: Reusability[StandardStep] =
+    Reusability.caseClassExcept('config)
```

Comment:
There are a few reason for this is, computing `config` could potentially be slow as the map inside could be largish and we do it for each step (at least displayed steps).
Additionally the only reason ever for config to change is if you do a `Sync` and that will refresh the whole table anyway.
Finally in `StepsTable` we don't display the config. we show it on the `StepConfigTable`

Reply:
Thank you, it seems then that it's safe to have these global `Reusability` instances that ignore `config`.